R/db.R

```diff
@@ -279,107 +279,97 @@ setMethod("sqlCreateTable", "DB2/AIX64",
 
 # Microsoft SQL Server ---------------------------------------------------------
 
+#' SQL Server
+#'
+#' Details of SQL Server specific variations for odbc and DBI generics.
```

Comment:
```suggestion
#' Details of SQL Server methods for odbc and DBI generics.
```

If this is technically accurate, it feels clearer to me.